src/module-elasticsuite-core/Model/System/Message/WarningAboutClusterGhostIndices.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * DISCLAIMER
+ * Do not edit or add to this file if you wish to upgrade Smile Elastic Suite to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ElasticsuiteCore
+ * @author    Romain Ruaud <romain.ruaud@smile.fr>
+ * @copyright 2022 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+
+namespace Smile\ElasticsuiteCore\Model\System\Message;
+
+use Magento\Framework\Notification\MessageInterface;
+use Magento\Framework\UrlInterface;
+use Smile\ElasticsuiteIndices\Model\IndexStatsProvider;
```

Comment:
As discussed, since this uses a class from ElasticsuiteIndices module, you can move this message to the Indices module to not create a dependency.

Reply:
Done.